src/supertux/menu/video_system_menu.cpp

```diff
@@ -34,12 +36,15 @@ void
 VideoSystemMenu::refresh()
 {
   add_label(_("Select Video System"));
-  add_inactive(fmt::format(_("Used video system: {}"), VideoSystem::get_video_string(g_config->video)));
   add_hl();
 
-  std::vector<std::string> video_systems = VideoSystem::get_available_video_systems();
-  for(unsigned i = 0; i < video_systems.size(); i++)
-    add_entry(static_cast<int>(i), video_systems[i]);
+  const auto video_systems = VideoSystem::get_available_video_systems();
+  for (int i = 0; i < static_cast<int>(video_systems.size()); ++i)
+  {
+    const VideoSystem::Info& video_system = video_systems.at(i);
+    add_entry(static_cast<int>(video_system.value), video_system.value == g_config->video ? '[' + video_system.name + ']' : video_system.name)
```

Comment:
Use fmt::format (or fmt::format(fmt::runtime if that doesn't work
```suggestion
    add_entry(static_cast<int>(video_system.value), video_system.value == g_config->video ? fmt::format("[{}]", video_system.name) : video_system.name)
```